### DISCUSSION TITLE
Low Cost BT Mini Connector V2 Powerline Adapter Performance

### DISCUSSION THREAD

**Discussion by giscus** [benmepham/benmepham.github.io#9] [Blog Comments]
# Low Cost BT Mini Connector V2 Powerline Adapter Performance

tl;dr: See table for speed and latency tests.

https://bjm.me.uk/blog/bt-powerline/

**Comment by gursimran982** [comment]
Helpful review, any chance you could create a short guide on how you used iperf3 to obtain these results. I've never used iperf3 before and not sure how to start.

**Reply by benmepham** [reply]
Thanks! Good idea, I'll add it to my backlog of posts to write...

For now I'd suggest checking out [this guide](https://www.linode.com/docs/guides/install-iperf-to-diagnose-network-speed-in-linux/) and the [docs](https://iperf.fr/iperf-doc.php#3doc).

**Comment by alexstrong29** [comment]
I have very fast internet shown in this photo 
https://share.icloud.com/photos/09aPP6rTYALvaamhgZJn1hu1w

And I have three mini connectors, one connected to the router and two elsewhere in the house. I cannot seem to get a good connection speed from them. Shown in this photo. Any thoughts?

https://share.icloud.com/photos/0b3bLRw5NjSPTc74f02hc4CDg

**Reply by benmepham** [reply]
There are so many factors that can affect the throughput of powerline adaptors such as the quality of your home's electrical wiring and what circuit your adaptors are on.

Although it is unusual for them to be that slow.

Perhaps try testing them plugged in in the same room?

Sorry I couldn't have helped more!
